src/components/SendItemInTransit/SendItemInTransit.js

```diff
@@ -1,32 +1,39 @@
-import { FormattedMessage } from 'react-intl';
-
 import {
-  Pane,
-  Paneset,
-} from '@folio/stripes/components';
+  useState,
+} from 'react';
+import { useIntl } from 'react-intl';
 
-import NavigationMenu from '../NavigationMenu';
+import ConfirmItem from '../ConfirmItem';
+import ErrorModal from '../ErrorModal';
 
 import {
-  FILTER_PANE_WIDTH,
+  CONFIRM_ITEM_TYPES,
   getSendItemInTransitUrl,
 } from '../../constants';
 
 const SendItemInTransit = () => {
+  const intl = useIntl();
+  const [contentData] = useState([]);
+  const [isErrorModalOpen, setErrorModalOpen] = useState(false);
+  const onCloseErrorModal = () => (setErrorModalOpen(false));
+  const handleSubmit = () => {};
```

Comment:
Is it expected to have empty function here?

Reply:
Will be implemented in scope of other story https://folio-org.atlassian.net/browse/UIREQMED-9